Source/Blazorise/Components/FilePicker/FilePicker.razor.cs

```diff
@@ -45,10 +50,11 @@ protected override void OnInitialized()
             ElementContainerId = IdGenerator.Generate;
             LocalizerService.LocalizationChanged += OnLocalizationChanged;
 
+            Attributes ??= new();
```

Comment:
I would like to not user `Attributes` like this. Try to assign needed HTML attributes directly to the input element where needed.

Reply:
Removed